src/app/dropdown/dropdown.component.css

```diff
@@ -131,6 +131,12 @@ hr {
     padding-top: 10px;
 }
 
+@media screen and (max-width: 2560px) {
+    .dropdown-menu-box {
```

Comment:
@harshit98 Why is this happening? This was fixed by me in a previous PR. I think this is a result of #726. Please look into this in this PR and fix it.
![image](https://user-images.githubusercontent.com/20185076/29239467-071d11d6-7f6d-11e7-9ccf-8c4ded342414.png)